atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableRenderer.java

```diff
@@ -1296,89 +1319,96 @@ public boolean apply(IndexMetadata index) {
         });
     }
 
-    private static final Class<?>[] IMPORTS = {
-        Set.class,
-        List.class,
-        Map.class,
-        SortedMap.class,
-        Callable.class,
-        ExecutorService.class,
-        Multimap.class,
-        Multimaps.class,
-        Collection.class,
-        Function.class,
-        Persistable.class,
-        Hydrator.class,
-        Transaction.class,
-        NamedColumnValue.class,
-        ColumnValue.class,
-        BatchingVisitable.class,
-        RangeRequest.class,
-        Prefix.class,
-        BatchingVisitables.class,
-        BatchingVisitableView.class,
-        IterableView.class,
-        ColumnValues.class,
-        RowResult.class,
-        Persistables.class,
-        AsyncProxy.class,
-        Maps.class,
-        Lists.class,
-        ImmutableMap.class,
-        ImmutableSet.class,
-        Sets.class,
-        HashSet.class,
-        HashMultimap.class,
-        ArrayListMultimap.class,
-        ImmutableMultimap.class,
-        Cell.class,
-        Cells.class,
-        EncodingUtils.class,
-        PtBytes.class,
-        MoreObjects.class,
-        Objects.class,
-        ComparisonChain.class,
-        Sha256Hash.class,
-        EnumSet.class,
-        Descending.class,
-        AbortingVisitor.class,
-        AbortingVisitors.class,
-        AssertUtils.class,
-        AtlasDbConstraintCheckingMode.class,
-        ConstraintCheckingTransaction.class,
-        AtlasDbDynamicMutableExpiringTable.class,
-        AtlasDbDynamicMutablePersistentTable.class,
-        AtlasDbNamedPersistentSet.class,
-        AtlasDbNamedExpiringSet.class,
-        AtlasDbMutablePersistentTable.class,
-        AtlasDbMutableExpiringTable.class,
-        AtlasDbNamedMutableTable.class,
-        ColumnSelection.class,
-        Joiner.class,
-        Entry.class,
-        Iterator.class,
-        Iterables.class,
-        Supplier.class,
-        InvalidProtocolBufferException.class,
-        Throwables.class,
-        ImmutableList.class,
-        UnsignedBytes.class,
-        Optional.class,
-        Collections2.class,
-        Arrays.class,
-        Bytes.class,
-        TypedRowResult.class,
-        TimeUnit.class,
-        CompressionUtils.class,
-        Compression.class,
-        Namespace.class,
-        Hashing.class,
-        ValueType.class,
-        Generated.class,
-        TableReference.class,
-        BatchColumnRangeSelection.class,
-        ColumnRangeSelections.class,
-        ColumnRangeSelection.class,
-        Iterators.class,
-    };
+    private List<String> getImports() {
+        List<String> imports = Lists.newArrayList();
+        if (useJava8Optionals) {
+            imports.add("java.util.Optional");
+        } else {
+            imports.add(Optional.class.getCanonicalName());
+        }
+        imports.addAll(
+        Arrays.asList(Set.class.getCanonicalName(),
```

Comment:
Did we need to switch from `ClassName.class` to calling `.getCanonicalName()` everywhere?